packages/gasket-plugin-analyze/lib/index.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
           name: 'analyze',
           description: 'Generate analysis report of webpack bundles',
           link: 'README.md#commands'
+        }],
+        configurations: [{
+          name: 'bundleAnalyzerConfig',
+          link: 'README.md#configuration',
+          description: 'This plugin uses [webpack-bundle-analyzer] to tune both browser and server analysis reports',
```

Comment:
If you were intended for this to link to webpack-bundle-analyzer it won't work without the reference. For these metadata descriptions, just keep them brief. 
```suggestion
          description: 'Tune both browser and server Webpack analysis reports',
```